modules/README.md

```diff
@@ -4,20 +4,45 @@ Modules should be entirely self-contained into a subdirectory with the following
 
     ModuleName/
 
-        ajax/         - This directory contains any AJAX helper script used by the module. They should
```

Comment:
It might be good to keep this with a deprecated warning, similar to what you added for the `templates/` folder below. Many module still rely on this directory.

Reply:
No one reading the document who doesn't know what it's already doing there should be pointed to it, though. The templates directory is a bit of a "softer" deprecation since it's not holding us back as much while the ajax directory adds an incredible amount of technical debt. Regardless, I added something to mention that it exists and is deprecated.